src/text/writer.rs

```diff
@@ -259,7 +291,21 @@ impl<W: Write> TextWriter<W> {
 
     /// Writes the provided byte array slice as an Ion clob.
     pub fn write_clob(&mut self, _value: &[u8]) -> IonResult<()> {
```

Comment:
Now that this function is defined, the `_value` variable is being used. The `_` name prefix is a hint to the compiler that we're ignoring this value intentionally; it's no longer being ignored, so you can remove the `_`.